specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -3174,6 +3178,29 @@
         }
       }
     },
+    "JupyterKernelConfig": {
+      "type": "object",
+      "description": "Jupyter kernel configuration",
+      "properties": {
+        "argv": {
+          "type": "array",
+          "description": "Argument to the the runtime",
+          "items": {
+            "type": "string",
+            "description": "argument values"
+          }
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of the kernel"
+        },
+        "language": {
+          "type": "string",
+          "description": "Language of the kernel"
```

Comment:
>Language of the kernel" [](http://example.com/codeflow?start=26&length=23)

better docs - speicified how? what are valid values?

Reply:
any value is valid, specifying as Enum is confusing.

Reply:
Yes, this is going to be a string in applications that want to display the kernel. It could be anything, including things we've never heard of, or even things that don't exist if the user wants to have an arbitrary division in the list or something.